src/app/page.tsx

```diff
@@ -23,22 +24,37 @@ export default async function HomeRedesign(props: {
   const chain = searchParams?.chain || 'mainnet';
   const api = searchParams?.api;
   const tokenPrice = await getTokenPrice();
-  const [stxPrice, recentBlocks, stackingCycle, initialTxTableData, mempoolStats, mempoolFee] =
-    await Promise.all([
-      getCurrentStxPrice(),
-      fetchRecentBlocks(chain, api),
-      fetchCurrentStackingCycle(chain, api),
-      fetchRecentUITxs(chain, api),
-      fetchUIMempoolStats(chain, api),
-      fetchMempoolFee(chain, api),
-    ]);
+  const [
+    stxPrice,
+    recentBlocks,
+    stackingCycle,
+    initialTxTableData,
+    mempoolStats,
+    sampleTxsFeeEstimate,
+  ] = await Promise.all([
+    getCurrentStxPrice(),
+    fetchRecentBlocks(chain, api),
+    fetchCurrentStackingCycle(chain, api),
+    fetchRecentUITxs(chain, api),
+    fetchUIMempoolStats(chain, api),
+    getSampleTxsFeeEstimate(chain as 'mainnet' | 'testnet', api),
```

Comment:
is there a better type we can use here?

Reply:
The thing is leather library expects a different chain type ('mainnet' | 'testnet' | 'devnet'...), so I had to cast our chain type to match.